test/unit/injector-deployment.bats

```diff
@@ -834,6 +834,14 @@ EOF
   local value=$(echo $object |
       yq -r 'map(select(.name=="AGENT_INJECT_MEM_REQUEST")) | .[] .value' | tee /dev/stderr)
   [ "${value}" = "64Mi" ]
+
+  local value=$(echo $object |
+      yq -r 'map(select(.name=="AGENT_INJECT_EPHEMERAL_REQUEST")) | .[] .value' | tee /dev/stderr)
+  [ "${value}" = "64Mi" ]
+
+  local value=$(echo $object |
+      yq -r 'map(select(.name=="AGENT_INJECT_EPHEMERAL_LIMIT")) | .[] .value' | tee /dev/stderr)
+  [ "${value}" = "128Mi" ]
```

Comment:
```suggestion
  local value=$(echo $object |
      yq -r 'map(select(.name=="AGENT_INJECT_EPHEMERAL_REQUEST")) | length' | tee /dev/stderr)
  [ "${value}" -eq 0 ]

  local value=$(echo $object |
      yq -r 'map(select(.name=="AGENT_INJECT_EPHEMERAL_LIMIT")) | length' | tee /dev/stderr)
  [ "${value}" -eq 0 ]
```

Reply:
@tomhjp I have removed this from the first test because we have made it an optional component.

Reply:
I think it's nice to have the assertion that it's not there in a test, because it's important that by default it doesn't get set.

Reply:
I tested with my suggestion and it fixed the test fwiw

Reply:
yeah, it is working when we don't add value, but if we add value then it fails as the length becomes 1